vars/utils.groovy

```diff
@@ -316,6 +316,31 @@ def loadJobTest() {
     }
 }
 
+def solverAnalyticsTest(){
+    def tuna_docker = docker.build("ci-tuna:${branch_id}", "--build-arg FIN_TOKEN=${FIN_TOKEN} --build-arg BACKEND=HIPNOGPU .")
+    tuna_docker.inside("--network host  --dns 8.8.8.8") {
+        checkout scm
+        env.TUNA_DB_HOSTNAME = "${db_host}"
+        env.TUNA_DB_NAME = "${db_name}"
+        env.TUNA_DB_USER_NAME = "${db_user}"
+        env.TUNA_DB_PASSWORD = "${db_password}"
+        env.gateway_ip = "${gateway_ip}"
+        env.gateway_port = "${gateway_port}"
+        env.gateway_user = "${gateway_user}"
+        env.PYTHONPATH = "${env.WORKSPACE}/tuna:${env.PYTHONPATH}"
+        env.PATH = "${env.WORKSPACE}/tuna:${env.PATH}"
```

Comment:
you probably want the PYTONPATH and the PATH to point to /tuna/SolverAnalytics right?